### PR TITLE
Add scoped keeper interface for interchain app

### PR DESCRIPTION
**Pull request by Anmol1696**
## Description

Add scoped keeper interface for interchain app

Part of: #2020

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (modules/core/exported) [\#1107](https://github.com/cosmos/ibc-go/pull/1107) Merging the `Header` and `Misbehaviour` interfaces into a single `ClientMessage` type.
 * (06-solomachine)[\#1906](https://github.com/cosmos/ibc-go/pull/1906/files) Removed `AllowUpdateAfterProposal` boolean as it has been deprecated (see 01_concepts of the solo machine spec for more details).
 * (07-tendermint) [\#1896](https://github.com/cosmos/ibc-go/pull/1896) Remove error return from `IterateConsensusStateAscending` in `07-tendermint`.
+* (apps/27-interchain-accounts) [\#2035](https://github.com/cosmos/ibc-go/pull/2035) Interchain host and client Keepers now expects a keeper which fulfills the expected `ScopedKeeper` interface for the capability keeper.
```
Comment (colin-axner) [inline CHANGELOG.md:62]:
```suggestion
* (apps/27-interchain-accounts) [\#2035](https://github.com/cosmos/ibc-go/pull/2035) Interchain accounts host and controller Keepers now expects a keeper which fulfills the expected `ScopedKeeper` interface for the capability keeper.
```

**Review by colin-axner — APPROVED** [review]
Thanks!

File: modules/apps/27-interchain-accounts/types/expected_keepers.go [hunk 2]
```diff
@@ -36,3 +36,10 @@ type PortKeeper interface {
 	BindPort(ctx sdk.Context, portID string) *capabilitytypes.Capability
 	IsBound(ctx sdk.Context, portID string) bool
 }
+
+type ScopedKeeper interface {
```
Comment (damiannolan) [inline modules/apps/27-interchain-accounts/types/expected_keepers.go:40]: Could we add a godoc? Just something short and sweet like the others would be perfect <3

**Review by damiannolan — APPROVED** [review]
Excellent, thank you! Just left one nit re. godoc! LGTM

File: modules/apps/27-interchain-accounts/types/expected_keepers.go (hunk 2 above)
Comment (Anmol1696) [reply modules/apps/27-interchain-accounts/types/expected_keepers.go:40]:
```suggestion
// ScopedKeeper defines the expected x/capability scoped keeper interface
type ScopedKeeper interface {
```

**Pull request merged**